jenkins/Machines.groovy

```diff
@@ -1,22 +1,27 @@
 def daint = [name: 'daint',
              archs: ['gpu', 'mc'],
-             cpus: ['gpu': 12, 'mc': 36],
              buildPath: '$XDG_RUNTIME_DIR/easybuild/build',
              unusePath: '/apps/daint/UES/jenkins/7.0.UP01/ARCH/easybuild/tools/modules/all:/apps/daint/UES/jenkins/7.0.UP01/ARCH/easybuild/modules/all',
              modulesProduction: '/apps/common/UES/jenkins/production/login',
              prefixProduction: '$APPS/UES/jenkins/7.0.UP01/ARCH/easybuild']
 
 def dom = [name: 'dom',
            archs: ['gpu', 'mc'],
-           cpus: ['gpu': 12, 'mc': 36],
            buildPath: '$XDG_RUNTIME_DIR/easybuild/build',
            unusePath: '/apps/dom/UES/jenkins/7.0.UP01/ARCH/easybuild/tools/modules/all:/apps/dom/UES/jenkins/7.0.UP01/ARCH/easybuild/modules/all',
            modulesProduction: '/apps/common/UES/jenkins/production/login',
            prefixProduction: '$APPS/UES/jenkins/7.0.UP01/ARCH/easybuild']
 
+def fulen = [name: 'fulen',
```

Comment:
Probably irrelevant for TSA?

Reply:
Yes, anyway the file `Machines.groovy` is addressed by #1510 that can be merged only after the recipes are available

Reply:
Removed `Machines.groovy` from this pull request: any change will be addressed in #1510